contributingGuides/KSv2.md

```diff
@@ -0,0 +1,16 @@
+# K2v2
+
+In Expensify we have an engineering prioritization process called "Kernel Scheduler v2" (KSv2). This system prioritizes tasks based on time priorities:
```

Comment:
Technically the system isn't just for the engineering team, but maybe you want to mention that because our contributors are currently all engineers?

Reply:
Oh good point, I took that text from the manifesto, but since it's used for also non-engineers I think makes sense to re-write that part to something like:

```
In Expensify, we have a task prioritization process called "Kernel Scheduler v2" (KSv2) that organizes tasks based on their time urgency:
```